pkg/yurthub/cachemanager/cache_manager.go

```diff
@@ -146,21 +146,27 @@ func (cm *cacheManager) queryListObject(req *http.Request) (runtime.Object, erro
 		return nil, err
 	}
 
-	var gvk schema.GroupVersionKind
 	var kind string
 	objs, err := cm.storage.List(key)
-	if err != nil {
-		return nil, err
-	} else if len(objs) == 0 {
-		gvk, err = serializer.UnsafeDefaultRESTMapper.KindFor(schema.GroupVersionResource{
+	// storage.ErrStorageNotFound indicates that there is no corresponding resource locally,
+	// and will determine whether to return an empty list structure according to whether GVK is registered or not
+	if err == storage.ErrStorageNotFound {
```

Comment:
1. storage.ErrStorageNotFound does not indicates empty list object on local disk, and indicates an error that request resource is not cached in local disk, so we need to return error here. 
2. and `len(objs)==0` indicates empty list object on local disk.

Reply:
Thanks for reminding me. I ignored this situation. The PR submitted recently corrected this mistake